packages/appstore/src/components/wallet-account/wallet-account.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react';
+import { Authorize } from '@deriv/api-types';
+import { Localize, localize } from '@deriv/translations';
+import { Icon, Money } from '@deriv/components';
+import { Text, Accordion, Button } from '@deriv/ui';
+import { ArrayElement } from 'Types';
+import WalletCard from 'Components/wallet';
+import WalletActionButton from 'Components/wallet-action-button';
+import AppsLauncher from 'Components/app-launcher';
+import { observer } from 'mobx-react-lite';
+
+type WalletAccountProps = {
+    account: ArrayElement<Required<Authorize>['account_list']>;
+};
+
+const WalletAccount = ({ account }: WalletAccountProps) => {
+    const isAccountVirtual = account.is_virtual === 1;
+
+    return (
+        <Accordion
+            expand_section={false}
+            elevation_type='xs'
+            onToggle={() => {
+                /**do something */
```

Comment:
If you want to add this later, you can add a TODO

Reply:
since `onToggle` is an required for `Accordion` I just added something to avoid TS error.